### ISSUE TITLE
Docs: hardware setup

### ISSUE THREAD

**Issue by mangtronix** [tve/lx16a-servo#1]
Hi,

What hardware setup are you using to connect the ESP32 to the servos? Is it as simple as wiring the ESP32 serial pin to the data pin on the first LX16A?

Looks like a nice library you've made. Thanks!

**Comment by tve** [comment]
Yes, I wired it straight. I don't remember whether I checked the logic level, i.e., whether it goes above 3.3V.
Let me know if you run into any issues.

**Comment by chefseg** [comment]
Do you have an electrical schematic of how this should be wired to the ESP32? I seem to be having issues communicating with the servo via ESP32.
